### PR TITLE
[BI-1085] - Germplasm import user input

### PR DESCRIPTION
**Pull request by ctucker3**
# Description
- Adds user inputs to the germplasm import during preview
- Adds user inputs to the brapi importer during preview
- Includes display of error banner if the list name is not provided. 

# Dependencies
- bi-api BI-1085

# Testing
- See bi-api BI-1085 for testing details.

### CODE REVIEW

**Review by dmeidlin — APPROVED** [review]

**Review by timparsons — CHANGES_REQUESTED** [review]
Happy path testing worked as expected.  I did try testing clicking the "Confirm" button with the import group name empty, and there was an error message of "An unknown error has occurred when uploading your import.", but the field in error wasn't highlighted.

File: src/views/import/ImportGermplasm.vue [hunk 1]
```diff
@@ -38,6 +42,31 @@
...
+        <form
+            class="new-form"
+            novalidate="true"
+        >
+          <p>The following information is required in order to keep a record of your import history.</p>
+          <div class="columns">
+            <div class="column is-one-quarter">
+              <BasicInputField
+                  v-model="germplasmList.germplasmListName"
+                  v-bind:field-name="'Import Group Name'"
```
Comment (timparsons) [inline src/views/import/ImportGermplasm.vue:55]: The wireframe has this labeled as "List Name"

Comment (ctucker3) [reply src/views/import/ImportGermplasm.vue]: fixed

**Comment by ctucker3** [comment]
> I did try testing clicking the "Confirm" button with the import group name empty, and there was an error message of "An unknown error has occurred when uploading your import.", but the field in error wasn't highlighted.

The error banner now shows this:
import_missing_field_error

**Review by timparsons — APPROVED** [review]

**Pull request merged**